mobile/bindings.go

```diff
@@ -93,9 +93,11 @@ func Start(extraArgs string, rpcReady Callback) {
 	// We call the main method with the custom in-memory listener called by
 	// the mobile APIs, such that the grpc server will use it.
 	cfg := lnd.ListenerCfg{
-		RPCListener: &lnd.ListenerWithSignal{
-			Listener: lightningLis,
-			Ready:    rpcListening,
+		RPCListeners: []*lnd.ListenerWithSignal{
+			&lnd.ListenerWithSignal{
```

Comment:
I think the type definition can be omitted here since it's already in the slice declaration.
Also this can be made a bit more compact by using the `{{` notation (though that is subject to personal taste):

```
		RPCListeners: []*lnd.ListenerWithSignal{{
			Listener: lightningLis,
			Ready:    rpcListening,
		}},
```